modules/telco-ran-managed-cluster-resources.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-du-overview.adoc
+
+:_content-type: REFERENCE
+[id="telco-ran-managed-cluster-resources_{context}"]
+= Worker node cluster resources
+
+. {product-title} CPU usage is less than 4000 mc and is pinned to reserved cores, including both hyperthreads for each reserved core.
```

Comment:
It's hard to tell if the sequence is important here. If not, they should be an unordered list per ISG